### PR TITLE
PUBDEV-8989 expose auto rebalance parameter

### PR DESCRIPTION
**Pull request by valenad1**
https://h2oai.atlassian.net/browse/PUBDEV-8989

I have exposed auto_rebalance parameter. It is available for testing now.

### CODE REVIEW

File: h2o-algos/src/main/java/hex/tree/gbm/GBM.java [hunk 1]
```diff
@@ -1522,4 +1522,8 @@ public PojoWriter makePojoWriter(Model<?, ?, ?> genericModel, MojoModel mojoMode
             LinkFunctionFactory.getLinkFunction(gbmMojoModel._link_function));
   }
 
+  @Override
+  protected void raiseReproducibilityWarning(String datasetName, int chunks) {
+    warn("auto_rebalance", "Rebalancing " + datasetName  + " dataset into " + chunks + " chunks. This model won't be reproducible on the different hardware configuration.");
+  }
```
Comment (wendycwong) [inline h2o-algos/src/main/java/hex/tree/gbm/GBM.java:1528]: This is nice!

**Review by wendycwong — APPROVED** [review]
Thank you!

File: h2o-algos/src/main/java/hex/tree/gbm/GBM.java (hunk 1 above)
Comment (valenad1) [reply h2o-algos/src/main/java/hex/tree/gbm/GBM.java:1528]: 💯

**Comment by h2o-ops** [comment]
**JIRA Issue Details**

Jira Issue: PUBDEV-9011
Assignee: Adam Valenta
Reporter: Exalate
State: Closed
Fix Version: N/A
Attachments: N/A
Development PRs: N/A

**Pull request merged**